pkgs/applications/science/math/cemu/default.nix

```diff
@@ -38,8 +39,13 @@ mkDerivation rec {
     libpng_apng
   ];
 
+  patches = [
+    ./stdexcept.patch
```

Comment:
nothing in this patch looks nixos specific. is there a way to upstream it?

Reply:
I just pushed changes that build without patching, by using gcc9Stdenv.

Reply:
I'm sure upstream could be fixed to build with stdenv. I'm not personally interested in using this package or contributing upstream for this package and it's dependencies (the patch touched submodules too).
My interest is in getting it to build again for ZHF.